luis-eduardo-salcedo/Tarea-11/router/src/App.js

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import  {Route, Routes} from 'react-router-dom';
+import "./App.css";
+import Home from "./components/home";
+import TaskOne from "./pages/taskOne";
+import TaskTwo from "./pages/taskTwo";
+import TaskThree from "./pages/taskThree";
+import TaskFour from "./pages/taskFour";
+import TaskFive from "./pages/taskFive";
+import TaskSix from "./pages/taskSix";
+import TaskSeven from "./pages/taskSeven";
+import TaskTen from "./pages/taskTen";
+
+export default function App() {
+    return (
+        <main className="app">
+            <Routes>
+                <Route path="/" element={<Home />} />
```

Comment:
@luisSalcedo8 , this route list should be dynamic and build from an Array using map functions